Makefile

```diff
@@ -3,12 +3,12 @@ NAME := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))
 VERSION := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["version"])')
 MANIFEST := build/collections/ansible_collections/$(NAMESPACE)/$(NAME)/MANIFEST.json
 
-ROLES := $(wildcard roles/*)
+ROLES := $(wildcard roles/*/*/*) $(wildcard roles/*/*)
```

Comment:
This is kind of lying. How about:

```suggestion
ROLES := $(wildcard roles/*)
ROLE_FILES := $(foreach ROLE,$(ROLES),$(wildcard $(ROLE)/*) $(wildcard $(ROLE)/*/*))
```

Reply:
BTW:
https://github.com/theforeman/foreman-ansible-modules/blob/dbd8ab270cac9d69f8b1cd1bd1a03e7ffb078260/Makefile#L10

Reply:
thank you!